Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md

```diff
@@ -0,0 +1,130 @@
+{"sequenceNumber":"0","output":""}</Data>
+  </EventData>
+</Event>
+```
+
+Example event message from Microsoft-ServiceFabric/Admin event log:
+
+```xml
+Log Name:      Microsoft-ServiceFabric/Admin
+Source:        Microsoft-ServiceFabric
+Date:          11/22/2020 14:30:52
+Event ID:      4173
+Task Category: Transport
+Level:         Warning
+Keywords:      Default
+User:          NETWORK SERVICE
+Computer:      nt0000001
+Description:
+authorization failure: CertificateNotMatched
+Event Xml:
+<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
+  <System>
+    <Provider Name="Microsoft-ServiceFabric" Guid="{cbd93bc2-71e5-4566-b3a7-595d8eeca6e8}" />
+    <EventID>4173</EventID>
+    <Version>2</Version>
+    <Level>3</Level>
+    <Task>16</Task>
+    <Opcode>0</Opcode>
+    <Keywords>0x8000000000000001</Keywords>
+    <TimeCreated SystemTime="2020-11-22T19:30:52.3311685Z" />
+    <EventRecordID>3087928</EventRecordID>
+    <Correlation />
+    <Execution ProcessID="15852" ThreadID="13412" />
+    <Channel>Microsoft-ServiceFabric/Admin</Channel>
+    <Computer>nt0000001</Computer>
+    <Security UserID="S-1-5-20" />
+  </System>
+  <EventData>
+    <Data Name="id">2173b6ea5d0</Data>
+    <Data Name="error">2147949758</Data>
+  </EventData>
+</Event>
+```
+
+## Cause
+
+DigiCert introduced a new CA which reuses the signing key of an existing and still-valid CA. This means there are 2 different CA certificates in circulation, and either can be included in the chain built for a certificate signed by this shared key. Existing certificates declared in SF clusters by subject with issuer pinning are at risk of spontaneously failing validation. This PKI/set of CAs is not restricted to a given cloud.
+
+## Impact
+
+Issue can cause either one or more nodes to stop participating in cluster or for entire cluster to stop functioning.
+
+## Mitigation
+
+If an upgrade is unfeasible or the cluster is already affected:
+
+- Install the CA cert with Thumbprint 626d44e704d1ceabe3bf0d53397464ac8080142c in the LocalMachine\Disallowed store in certlm.msc, "Local Computer\Untrusted certificates"  
+- Issuer/ Intermediate certficate 626d44e704d1ceabe3bf0d53397464ac8080142c can be downloaded from https://www.digicert.com/kb/digicert-root-certificates.htm
+
+example command once .crt file has been downloaded:  
+
+```cmd
+certutil -addstore -enterprise Disallowed .\DigiCertSHA2SecureServerCA-2.crt
+```
+
+or  
+
+- If you have multiple certificates with the same CN from authorized issuers, fall back to one not signed by the shared key by deleting the conflicting certificate and let the cluster choose another certificate. This change however has risks and should be tested.
```

Comment:
We should remove the option of falling back to another cert. It's high risk, will require rebooting nodes, so in that case the first mitigation is better/less confusing/more reliable.